### ISSUE TITLE
postrelease should not check for towncrier fragments

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/zestreleaser.towncrier#16]
I just released `zestreleaser.towncrier 1.0.3`.  (Ah, should have been 1.1.0...) During postrelease, the new check was needlessly done:

```
INFO: Starting postrelease.
Found 0 towncrier newsfragments with recognized extension.

WARNING: No towncrier newsfragments found.
The changelog will not contain anything interesting.

Do you want to continue anyway? (y/N)?  y
INFO: Doing dry-run of towncrier to see what would be changed: /Users/maurits/community/zestreleaser.towncrier/bin/towncrier --draft --version t.b.d. --yes
...
```

We should not run the check during postrelease.
(A warning that some snippets have been left behind could be okay, but should be superfluous.)

**Comment by mauritsvanrees** [comment]
For good measure, I have created a branch 1.0.x and rereleased 1.0.2 as 1.0.4, to avoid the new features.
I have rereleased master as 1.1.0.

**Comment by mauritsvanrees** [comment]
Fix released in 1.2.0.
